tasks/main.yml

```diff
@@ -31,6 +31,6 @@
   setup: filter=ansible_local
   when: facts.changed
 
-- include: install.yml
+- import_tasks: install.yml
```

Comment:
what is the difference here with `include-role`?

Reply:
`include_role` is dynamic, so it will do the inclusion every time the line is executed, `import_tasks` is static, so it will do the inclusion once, during pre-processing.